src/components/InpuField.css

```diff
@@ -1,21 +1,50 @@
-.inputfield {
+.Add,
```

Comment:
class names should be of lowercase

```suggestion
.add
```